exposee-app/server.js

```diff
@@ -65,7 +113,21 @@ app.post('/videos', async (req, res) => {
   res.status(500).json({ message: error.message });
   }
 });
-
+app.post('/broadcast', async (req, res)=> {
```

Comment:
broadcast doesn't seem to have anything to do with profile picture, so I would have separated into it's own PR. However since you tested it, it's OK to keep this time.